packages/database/src/migrations/20210804054649-CovidSamoaReinstateHouseholdOverlay-modifies-data.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+import { codeToId, insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+const mapOverlayGroupCode = 'COVID19_Samoa';
+const mapOverlayId = 'WS_COVID_Household_Vaccination_Status';
+const reportCode = 'WS_COVID_Household_Vaccination_Status';
+
+const mapOverlayGroupRelation = groupId => ({
+  id: generateId(),
+  map_overlay_group_id: groupId,
+  child_id: mapOverlayId,
+  child_type: 'mapOverlay',
+  sort_order: 4,
+});
+
+exports.up = async function (db) {
+  const mapOverlayGroupId = await codeToId(db, 'map_overlay_group', mapOverlayGroupCode);
+  await insertObject(db, 'map_overlay_group_relation', mapOverlayGroupRelation(mapOverlayGroupId));
+  await db.runSql(`
+    update report
+        set config = jsonb_set(config, '{transform,1,where}', '"notEq($row.HVS_covidstatus, ''Fully vaccinated'')"')
+        where code = '${reportCode}';
+  `);
+  // add fetchOnLevel config
+  await db.runSql(`
+    update "mapOverlay" 
+      set "presentationOptions" = "presentationOptions" || '{"fetchOnlyDisplayLevel": true}'
```

Comment:
Should we maybe not do the `fetchOnLevel` stuff here and save that for when we actually implement the performance stuff?

Reply:
Good call! I just missed taking it out.